### PR TITLE
wallet misc

### PR DESCRIPTION
**Pull request by turadg**
## Description

- test(agops): smoke test
- chore(smart-wallet): more logging
- build(smart-wallet): build bundles
- fix(agoric-cli): perf watch
- fix(agoric-cli): psm giveMinted
- chore: safer offerResults publishing

### Security Considerations

Stops publishing offerResult for copyRecord. Publishes simple types.

### Documentation Considerations

--
### Testing Considerations

--

### CODE REVIEW

File: packages/smart-wallet/src/offers.js [hunk 1]
```diff
@@ -43,7 +44,7 @@ export const makeOfferExecutor = ({
   onStatusChange,
   onNewContinuingOffer,
 }) => {
-  const { invitationFromSpec, lastOfferId, purseForBrand } = powers;
+  const { invitationFromSpec, lastOfferId, logger, purseForBrand } = powers;
```
Comment (dckc) [inline packages/smart-wallet/src/offers.js:47]: :+1: ocap-happy logging

File: packages/smart-wallet/src/smartWallet.js [hunk 2]
```diff
@@ -343,7 +343,9 @@ const behavior = {
      */
     async executeOffer(offerSpec) {
       const { facets, state } = this;
+      /** @type {State} */
       const {
+        address,
```
Comment (dckc) [inline packages/smart-wallet/src/smartWallet.js:348]: hm. I missed something somewhere. A wallet knows the address it's associated with? Is that necessary? It sets off my ACL spidey-sense... it feels vagely like [caller](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/caller) or `msg.sender` in solidity.

File: packages/smart-wallet/src/offers.js [hunk 3]
```diff
@@ -134,15 +143,19 @@ export const makeOfferExecutor = ({
...
+          numSatisfied => {
+            logger.info(id, 'numSatisfied', numSatisfied);
+            if (numSatisfied === 0) {
+              updateStatus({ numWantsSatisfied: 0 });
+            }
+            updateStatus({
+              numWantsSatisfied: numSatisfied,
+            });
+          },
+          handleError,
```
Comment (dckc) [inline packages/smart-wallet/src/offers.js:157]: The commit message doesn't mention this. (not critical)

**Review by dckc — APPROVED** [review]
:+1:

File: packages/smart-wallet/src/smartWallet.js (hunk 2 above)
Comment (turadg) [reply packages/smart-wallet/src/smartWallet.js:348]: Yeah, I had the same tingle. It's not strictly necessary. We could take it out by passing two other things in instead:
- wallet logger
- wallet storageNode

But it has to persist through init and there may be some churn with durability, so I'll punt until then: https://github.com/Agoric/agoric-sdk/issues/5894

**Pull request merged**